language/de_x_sie/email/user_forgot_password.txt

```diff
@@ -0,0 +1,14 @@
+Subject: Benutzerkonto-Passwort zurücksetzen
+
+Hallo {USERNAME},
```

Comment:
in der de_x_sie nutzen wir fast immer "Guten Tag" statt "Hallo", ich würde das hier auch so machen und die übrigen "Hallo" ggf. nachziehen.

Reply:
+1